app/markets/ActivityTable.tsx

```diff
@@ -80,35 +83,42 @@ export const ActivityTable = ({ id }: { id: string }) => {
           <LoadingSkeleton />
         ) : (
           <TableBody className="text-base font-semibold">
-            {activities.map(activity => (
-              <TableRow key={activity.transactionHash}>
-                <TableCell className="pl-4 text-text-high-em">
-                  <a
-                    href={getGnosisAddressExplorerLink(activity.user.id)}
-                    className="hover:underline"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    {shortenAddress(activity.user.id)}
-                  </a>
-                </TableCell>
-                <TableCell>
-                  <Tag
-                    size="xs"
-                    colorScheme={getTagColorScheme(activity.transactionType)}
-                    className="w-fit uppercase"
-                  >
-                    {activity.transactionType}
-                  </Tag>
-                </TableCell>
-                <TableCell className="truncate text-text-high-em">
-                  {formatEtherWithFixedDecimals(activity.sharesOrPoolTokenAmount)}
-                </TableCell>
-                <TableCell className="text-nowrap pr-4 text-right text-text-low-em">
-                  {formatDateTime(activity.creationTimestamp)}
-                </TableCell>
-              </TableRow>
-            ))}
+            {activities?.map(activity => {
+              const outcomes = activity.fpmm.outcomes;
+              const outcomeIndex = activity.outcomeIndex;
+
+              if (!outcomes || !outcomeIndex) return null;
+
+              return (
+                <TableRow key={activity.transactionHash}>
+                  <TableCell className="pl-4 text-text-high-em">
+                    <a
+                      href={getGnosisAddressExplorerLink(activity.creator.id)}
+                      className="hover:underline"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      {shortenAddress(activity.creator.id)}
+                    </a>
+                  </TableCell>
+                  <TableCell>
+                    <Tag
+                      size="xs"
+                      colorScheme={getTagColorScheme(outcomes[outcomeIndex], outcomes)}
+                      className="w-fit uppercase"
+                    >
+                      {outcomes[outcomeIndex]}
```

Comment:
Let's declare an Outcome instance and use the outcome symbol here. We shorten outcomes so they can fit in components like this.

Reply:
I didn't fully understand this comment, sorry, could you elaborate a bit on what would you like to see happening here?

Reply:
Commented here: https://github.com/SwaprHQ/presagio/pull/63/files#r1677534154